gsuite-device-update-notifier/main.js

```diff
@@ -1,5 +1,5 @@
-var MINIMUM_ANDROID_VERSION = new Date(2019, 9, 5);
-var ANDROID_LAST_UPDATE = new Date(2019, 9, 9);
+var MINIMUM_ANDROID_VERSION = new Date(2019, 10, 5);
+var ANDROID_LAST_UPDATE = new Date(2019, 10, 4);
```

Comment:
Shouldn't this be November instead of October?

Reply:
Javascript dates:
```
> new Date(2019, 10, 4)
Mon Nov 04 2019 00:00:00 GMT-0500 (Eastern Standard Time)
```